modules/assim.batch/R/pda.emulator.R

```diff
@@ -55,10 +64,10 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     con <- NULL
   }
 
-  bety <- PEcAn.visualization::betyConnect("~/pecan/web/config.php")
+  bety <- betyConnect("~/pecan/web/config.php")
```

Comment:
what's the final verdict on this?

I can do
```
bety <- src_postgres(dbname = settings$database$bety$dbname, 
               host = settings$database$bety$host, 
               user = settings$database$bety$user, 
               password = settings$database$bety$password)
```
for now and open an issue, so that @bcow can look later?